travel-event-host-app/src/models/event.ts

```diff
@@ -21,6 +21,6 @@ interface Event {
   };
   startDate: Date;
   endDate: Date;
-  category: Category[];
+  categories: string[];
```

Comment:
Was there a specific reason to change this from `Category[]` to `string[]` ?

Reply:
Yeah, I don't think the enum will help, my first impression was use numbers to save the categories on Mongodb, and saving string categories no Category type